mist/worker/src/main/scala/io/hydrosphere/mist/worker/runners/ArtifactDownloader.scala

```diff
@@ -55,21 +57,24 @@ case class HttpArtifactDownloader(
     val uri = s"http://$masterHttpHost:$masterHttpPort/v2/api/artifacts/${encode(filePath)}/sha"
     val request = HttpRequest(method = HttpMethods.GET, uri = uri)
     for {
-      r        <- Http().singleRequest(request)
-      resp     =  checkResponse(uri, r)
-      checksum <- Unmarshal(resp.entity).to[String]
+      r <- Http().singleRequest(request)
+      resp = checkResponse(uri, r)
+      checksum <- Unmarshal(resp.entity.withSizeLimit(maxArtifactSize)).to[String]
```

Comment:
Sha has fixed size - `withSizeLimit` usage seems useless there.

Reply:
Copy-paste mistake, ok I'll remove it:)